herbert_core/utilities/misc/hlp_serial_size.m

```diff
@@ -14,6 +14,16 @@
 %
 %   dispatch according to type
 %
+
+% Always initialise first
+global types_size;
+if isempty(types_size)
```

Comment:
hlp_serialize is in theory independent, so I wouldn't count that along with this implementation. I also agree it would be nice to have a global constant, but I'm not sure how to do that cleanly. I guess the way in MATLAB would be through objects? But I'm not quite sure. Any advice would be appreciated.